deny.toml

```diff
@@ -59,10 +59,14 @@ license-files = [
 # Deny multiple versions or wildcard dependencies.
 multiple-versions = "deny"
 wildcards = "deny"
+# failure-server is an internal package used only in testing that we do not want to publish to
+# crates.io. when including it in tough's dev dependencies, it is necessary to do so without a
+# version number or else cargo publish will fail. the lack of a version number causes cargo deny
+# to ban failure-server as a wildcard-versioned dependency. the following line allows us to refer
+# to our own internal packages without a version number.
+allow-wildcard-paths = true
 
 skip = [
-    # http-test is using an old version of bstr
```

Comment:
Is this not applicable any more?

Reply:
I need to add it back, actually. For reasons I don't understand I have to use `make check-licenses` locally instead of using `cargo deny` directly.